indexer-app/src/cmr/indexer/data/index_set.clj

```diff
@@ -293,6 +293,8 @@
           :start-date                     (m/stored m/date-field-mapping)
           :end-date                       (m/stored m/date-field-mapping)
```

Comment:
I'm confused. If we have :temporal-ranges, do we still need :start-date :end-date here?